src/test/java/org/jenkinsci/plugins/hello/HelloWorldBuilderTest.java

```diff
@@ -1,5 +1,29 @@
 package org.jenkinsci.plugins.hello;
 
+/*
```

Comment:
its not really the top of the file :-)  
It is normal to put this first before anything else (including the package statement)